dashboard/app/models/unit.rb

```diff
@@ -1940,4 +1940,8 @@ def has_ai_tutor_level?
     initiative = get_course_version&.course_offering&.marketing_initiative
     TEACHER_FEEDBACK_INITIATIVES.include? initiative
   end
+
+  private def sorted_user_facing_resources(resources)
+    resources.filter {|r| r.embeddability_type != SharedConstants::RESOURCE_EMBEDDABILITY_OPTIONS[:EMBED_ONLY][:value]}.sort_by(&:name).map(&:summarize_for_resources_dropdown)
```

Comment:
You have numerous filters on the `embeddability_type` added in this PR. Let's put `show_in_resource_ui?` and `embed_in_ai_ta?` helper methods in the `Resource` class to make these filters easier to read.

```suggestion
    resources.filter(&:show_in_resource_ui?).sort_by(&:name).map(&:summarize_for_resources_dropdown)
```